database/migrations/20180519000006-create-prediction.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
         game_id: foreignKey('hp_game', false),
         risk_id: foreignKey('hp_risk', false),
       }))
+      .then(() => queryInterface.addIndex('hp_prediction', {
+        fields: ['user_id', 'game_id'],
+        unique: true,
+      }))
```

Comment:
En effet, la doc a l'air de dire que pour des FK qui ne consistituent pas une PK, l'index n'est pas fait automatiquement.

:+1: 

https://stackoverflow.com/questions/970562/postgres-and-indexes-on-foreign-keys-and-primary-keys